### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by TamaraDimov**
Requirements:

- [ ]  Implement the design from the sneak peek wireframes
- [ ]  I create an HTML view for user and post.
- [ ]  I create comment HTML file.

### CODE REVIEW

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      <% if local_assigns[:show_all_comments] %>
+        <ddiv class="post__counters">
+          Comments: <%= post.commentsCounter %>, Likes: <%= post.likesCounter %>
+        </ddiv>
+      <% end %>
+      <div>
+        <%= post.text.truncate(20) if truncated_message %>
+        <%= post.text unless truncated_message %>
+      </div>
```
Comment (MahmoudBakr23) [inline app/views/posts/_post.html.erb]: - [ ] Please make sure that the content rendered in the `posts/index` is the posts with the most recent 3 comments for each. 🙏

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<%= link_to user_post_path(post.author, post) do %>
+  <article class="post_view">
+  <div class="card mb-3" style="max-width: 540px;">
+    <div class="post__container">
+      <h3><%= post.title %></h3>
+
+      <% if local_assigns[:show_all_comments] %>
+        <ddiv class="post__counters">
+          Comments: <%= post.commentsCounter %>, Likes: <%= post.likesCounter %>
+        </ddiv>
```
Comment (MahmoudBakr23) [inline app/views/posts/_post.html.erb]: - [ ] Please fix the typo here in the `div` tag on line 8 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @TamaraDimov 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done working with the controllers so far
- Nice job using partials in the views
- Descriptive PR and README
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️

- [ ] Kindly fully follow the given design wireframe and implement the styles in your app. 🙏 

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @TamaraDimov , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

Your project have an error while setting up, this error happens when creating comments in your seed.rb so make sure you fix this :pray: 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ibizugbe — APPROVED** [review]
Hi @TamaraDimov 👋🏼👋🏼👋🏼, 

approved


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] Typo error has been fixed on `_post.html.erb` as requested by the previous reviewer👍🏼.
- [x] The post/index now renders the 3 most recent posts👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**